### DISCUSSION TITLE
Does App Router Cache only work when deploying on Vercel?

### DISCUSSION THREAD

**Discussion by welschmoor** [vercel/next.js#52067] [App Router]
If I remember correctly, the cache of the old router when using getServerSideProps was only working when deploying on Vercel.
We deploy on Azure. Will the cache of the fetches that I run inside the server components of the new app router be cached if I deploy it anywhere or is it only limited to Vercel?

Also, can somebody tell me where this cache is stored? Can I find it inside the .next folder (or the folder that replaces .next folder, I think I may have read a new dir is used for app router)?

**Comment by nabildroid** [comment]
If you are running your app in serverelss environment you can't use next caching out-of-box
Because next stores all caches within (in run time generated) .next folder

How ever if you attached a presistence disk to the serverelss environment and update the config to point to that disk you can achieve caching across coldboots

**Reply by welschmoor** [reply]
I know too little about "serverless", all I can say, we have 4 Kubernetes Pods running our frontend

**Reply by nabildroid** [reply]
Attach them to a shared volume
